projects/veRocket/index.js

```diff
@@ -1,38 +1,17 @@
 const { get } = require('../helper/http')
-const { toUSDTBalances } = require('../helper/balances');
 
 const baseUrl = "https://api-v1.verocket.com";
 const tvlApy = `${baseUrl}/dex/overall/lp_volume`
-const vetPrice = `${baseUrl}/price/vet`
-
-
-const getVetPrice = async () => {
-  return (await get(vetPrice)).data.usd
-}
 
 async function tvl() {
   balances = 0;
   const response = (await get(tvlApy)).data;
-
-  // get vet price
-  const vetPrice = await getVetPrice();
-
-  // fetch latest TVL from the list of 30 days TVL
-  tvlObjects = response[29].items;
-
-  tvlObjects.forEach(el => {
-    balances += el.eq_vet * vetPrice;
-  });
-
-  return toUSDTBalances(balances)
+  return { 'vechain': +response[response.length - 1].eq_vet}
```

Comment:
@g1nt0ki You are returning the TVL in terms of VET tokens. Where does it get converted into USD? And thanks for the code refactoring, happy to learn it from you.

Reply:
we fetch token prices from the server and convert it to USD, you can check `computeTVL` method in @defillama/sdk  :)